presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreenState.kt

```diff
@@ -1,10 +1,12 @@
 package me.androidbox.presentation.agenda.screen
 
+import me.androidbox.domain.authentication.model.Event
 import java.time.ZoneId
 import java.time.ZonedDateTime
 
 data class AgendaScreenState(
     val displayMonth: String = ZonedDateTime.now(ZoneId.systemDefault()).month.toString(),
     val usersInitials: String = "",
-    val shouldOpenDropdown: Boolean = false
+    val shouldOpenDropdown: Boolean = false,
+    val listOfEventDetail: List<Event> = emptyList()
```

Comment:
If an attribute is a list, no need to add `listOf` to the name

Just `events` should be more than enough

It's like having a boolean, and name it
booleanShouldHide

Reply:
@MKiperszmid updated